test/unit/meta/concepts/native_simd_for_abi.cpp

```diff
@@ -66,7 +66,7 @@ TTS_CASE_TPL( "Check that wide<T,N,ppc*> does not satisfy native_simd_for_abi wi
   TTS_EXPECT_NOT((eve::native_simd_for_abi<logical<wide<T>> , eve::arm_64_ , eve::arm_128_  > ));
   TTS_EXPECT_NOT((eve::native_simd_for_abi<logical<wide<T>> , eve::x86_128_ , eve::x86_256_, eve::x86_512_ > ));
 };
-#elif defined(SPY_SIMD_IS_ARM_FIXED_SVE)
+#elif defined(SPY_SIMD_IS_ARM_FIXED_SVE) || defined(SPY_SIMD_IS_ARM_FIXED_SVE2)
```

Comment:
should we have one macro? @jfalcou

Reply:
Some global one ? EVE could add one if you think it improves ergonomics.
Some `EVE_SIMD_IS_ANY_FIXED_SVE` ?

Reply:
Maybe